mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -1442,6 +1442,53 @@ resource "google_container_node_pool" "np" {
 `, cluster, np, placementType)
 }
 
+func TestAccContainerNodePool_threadsPerCore(t *testing.T) {
+	t.Parallel()
+
+	cluster := fmt.Sprintf("tf-test-cluster-%s", RandString(t, 10))
+	np := fmt.Sprintf("tf-test-nodepool-%s", RandString(t, 10))
+
+        VcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderFactories(t),
+		CheckDestroy: testAccCheckContainerNodePoolDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerNodePool_threadsPerCore(cluster, np, 1),
+			},
+			{
+				ResourceName:      "google_container_cluster.cluster",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccContainerNodePool_threadsPerCore(cluster, np string, threadsPerCore int) string {
+	return fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
```

Comment:
Do you mind adding a top-level `node_config` block and setting the feature there too? That'll enable it in the default node pool.

`node_config.go.erb` is reused in 3 places, I tend to make sure we're using the field in 2 of those.

Reply:
done